Helper/Price.php

```diff
@@ -210,25 +210,34 @@ public function getProductPrice( // @codingStandardsIgnoreLine
             // We will use the SKU that has the lowest final price
             case ConfigurableType::TYPE_CODE:
                 $productType = $product->getTypeInstance();
-                $price = null;
+                $price = 0.0;
                 if ($productType instanceof ConfigurableType) {
                     $products = $this->nostoProductRepository->getSkus($product);
                     $skus = [];
                     $finalPrices = [];
+                    $outOfStockFinalPrices = [];
                     foreach ($products as $sku) {
-                        if ($sku instanceof Product
-                            && !$sku->isDisabled()
-                            && $sku->isAvailable()
-                        ) {
-                            $finalPrices[$sku->getId()] = $this->getProductPrice(
-                                $sku,
-                                $store,
-                                $inclTax,
-                                true
-                            );
+                        if ($sku instanceof Product) {
+                            if (!$sku->isDisabled() && $sku->isAvailable()) {
+                                $finalPrices[$sku->getId()] = $this->getProductPrice(
+                                    $sku,
+                                    $store,
+                                    $inclTax,
+                                    true
+                                );
+                            } else {
```

Comment:
How heavy is the getProductPrice()? should we skip this if there is at least one sku available?
something like : else if (!empty($finalPrices))

Reply:
Good point, i'll add an extra check